flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```

Comment:
Upstream seem to support building on x86_64. Any reason to restrict architecture here?

Reply:
Upstream provides only [x86_64 builds](https://github.com/loganmc10/m64p/releases) for linux and windows. Also, it uses the x86_64 build of the libvosk library as a pre-built binary for VRU emulation, so it would only work on x86_64 platforms. It might be possible to port it for other platforms, but I can't test it myself.

Reply:
> Upstream provides only [x86_64 builds](https://github.com/loganmc10/m64p/releases) for linux and windows. Also, it uses the x86_64 build of the libvosk library as a pre-built binary for VRU emulation, so it would only work on x86_64 platforms. It might be possible to port it for other platforms, but I can't test it myself.

There is an aarch64 wheel for it. Also you could build from source too.

What kind of build upstream provide is irrelevant when building from source.